test/runtests.cmd

```diff
@@ -334,13 +334,18 @@ goto :main
 
   set _BuildArchMapped=%_BuildArch%
   set _BuildTypeMapped=%_BuildType%
+  :: _NewBuildArchMapped and _NewBuildTypeMapped store new terms of the build and architecture
+  :: _NewBuildArchMapped: x86/x64 _BuildArchMapped:x86/amd64
+  :: _NewBuildTypeMapped: debug/test _BuildTypeMapped:chk/fre
+  set _NewBuildArchMapped=%_BuildArch%
+  set _NewBuildTypeMapped=%_BuildType%
 
   :: Map new build arch and type names to old names until rl test tags are
   :: updated to the new names
   if "%_BuildArchMapped%" == "x64" set _BuildArchMapped=amd64
   if "%_BuildTypeMapped%" == "debug" set _BuildTypeMapped=chk
   if "%_BuildTypeMapped%" == "test" set _BuildTypeMapped=fre
-  if "%_BuildTypeMapped%" == "codecoverage" set _BuildTypeMapped=fre
+  if "%_BuildTypeMapped%" == "codecoverage" set _BuildTypeMapped=fre set _NewBuildTypeMapped=test

```

Comment:
I haven't tried exercising this line yet, but is this actually valid syntax in cmd?

Shouldn't this be more like:

```
if "%_BuildTypeMapped%" == "codecoverage" (
  set _BuildTypeMapped=fre
  set _NewBuildTypeMapped=test
)
```